collect_app/src/main/res/raw/settings_schema.json

```diff
@@ -0,0 +1,339 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://getodk.com/settings.schema.json",
+  "title": "Settings",
+  "type": "object",
+  "properties": {
+    "project": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "icon": {
+          "type": "string"
+        },
+        "color": {
+          "type": "string"
+        }
+      }
+    },
+    "general": {
+      "type": "object",
+      "properties": {
+        "protocol": {
+          "type": "string",
+          "enum": [
+            "odk_default",
+            "google_sheets"
+          ]
+        },
+        "server_url": {
+          "type": "string"
+        },
+        "username": {
+          "type": "string"
+        },
+        "password": {
+          "type": "string"
+        },
+        "selected_google_account": {
+          "type": "string"
+        },
+        "google_sheets_url": {
+          "type": "string"
+        },
+        "appTheme": {
+          "type": "string",
+          "enum": [
+            "system",
+            "light_theme",
+            "dark_theme"
+          ]
+        },
+        "app_language": {
+          "type": "string"
+        },
+        "font_size": {
+          "type": "string",
+          "enum": [
+            "13",
+            "17",
+            "21",
+            "25",
+            "29"
+          ]
+        },
+        "navigation": {
+          "type": "string",
+          "enum": [
+            "swipe",
+            "buttons",
+            "swipe_buttons"
+          ]
+        },
+        "showSplash": {
+          "type": "boolean"
+        },
+        "splashPath": {
+          "type": "string"
+        },
+        "basemap_source": {
+          "type": "string",
+          "enum": [
+            "google",
+            "mapbox",
+            "osm",
+            "usgs",
+            "stamen",
+            "carto"
+          ]
+        },
+        "google_map_style": {
+          "type": "string",
+          "enum": [
+            "1",
+            "2",
+            "3",
+            "4"
+          ]
+        },
+        "mapbox_map_style": {
+          "type": "string",
+          "enum": [
```

Comment:
I think here you should have 6 values and you forgot `mapbox://styles/mapbox/streets-v11`?